modules/serverless-deprecated-removed-features.adoc

```diff
@@ -53,22 +53,25 @@ For the most recent list of major functionality deprecated and removed within {S
 .Deprecated and removed features tracker for {rosa-product-title}
 [cols="3,1,1,1",options="header"]
 |====
-|Feature |1.23 to 1.26|1.27|1.28
+|Feature |1.23 to 1.26|1.27|1.28|1.29
```

Comment:
This second table looks broken in the preview. The "1.29" cell is on the new line instead of in headers.

Reply:
Thanks! (it was actually a missing `,1` in `cols="3,1,1,1,1"`)